main/EditText/index.tsx

```diff
@@ -327,83 +322,69 @@ function EditText(props: Props): ReactElement {
                 : focused
                   ? [{ color: focusColor }, focusedLabelStyle]
                   : null,
-            ]}>
+            ]}
+          >
             {label}
           </StyledLabel>
-          {
-            leftElement
-              ? (
-                <StyledIcon style={[{ width: 40 }, leftElementStyle]}>
-                  {leftElement}
-                </StyledIcon>
-              ) : null
-          }
-          <StyledTextInput
-            {...textInputProps}
-            testID={testID}
-            autoCapitalize={autoCapitalize}
-            onFocus={(): void => {
-              setFocus(true);
-              if (onFocus) {
-                onFocus();
-              }
-            }}
-            onBlur={(): void => {
-              setFocus(false);
-              if (onBlur) {
-                onBlur();
-              }
-            }}
-            placeholder={placeholder}
-            placeholderTextColor={placeholderTextColor}
-            value={value}
-            style={textStyle}
-            onChangeText={onChangeText}
-            secureTextEntry={secureTextEntry}
-            onSubmitEditing={onSubmitEditing}
-            multiline={multiline}
-          />
-          {
-            rightElement
-              ? (
-                <StyledIcon style={[{ width: 40 }, rightElementStyle]}>
-                  {rightElement}
-                </StyledIcon>
-              ) : null
-          }
-          {
-            borderWidth
-              ? <StyledLine
-                style={[
-                  { borderWidth: borderWidth, borderRadius: inputContainerRadius, borderColor: borderColor },
-                  borderStyle,
-                  errorText
-                    ? { borderColor: errorColor }
-                    : focused
-                      ? [{ borderColor: focusColor }, { borderWidth: focusedBorderWidth }]
-                      : null,
-                  !leftElement
-                    ? { paddingLeft: 15 }
-                    : null,
-                  !rightElement
-                    ? { paddingRight: 15 }
-                    : null,
-                ]}
-              />
-              : null
-          }
```

Comment:
Don't we need this anymore?

Reply:
With this code, TextInput layout is little bit twisted in web.

Reply:
When I remove this code, It does't make any side effect in my sight.